hypha/apply/activity/management/commands/send_staff_email_digest.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import re
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from django.utils.translation import gettext as _
+
+from hypha.apply.activity.adapters import SlackAdapter
+from hypha.apply.activity.models import Message
+from hypha.apply.activity.options import MESSAGES
+from hypha.core.mail import send_mail
+
+logger = logging.getLogger(__name__)
+
+
+def slack_message_to_markdown(msg):
+    def to_href(match_obj):
+        if match_obj.group() is not None:
+            return f"[{match_obj.group(2)}]({match_obj.group(1)})"
```

Comment:
We prefer single quotes in all python code.

Reply:
Sure. I'll update this.